api/queries/community/threadConnection.js

```diff
@@ -44,10 +50,10 @@ export default async (root: DBCommunity, args: PaginationOptions, ctx: GraphQLCo
     channels = await getPublicChannelsByCommunity(id);
   }
 
-  // $FlowFixMe
   const threads = await getThreadsByChannels(channels, {
     first,
     after: lastThreadIndex,
+    sort: sort === 'NEW' ? 'new' : 'trending',
```

Comment:
Caps vs lowercase - then we could just say: { sort } if the sort argument is either `new` or `trending`

Reply:
GraphQL vs internal db query options